packages/components/organisms/f-header/src/components/FlagIcon.vue

```diff
@@ -18,14 +18,12 @@ import {
     FlagItRoundIcon as FlagIconIt,
     FlagLuRoundIcon as FlagIconLu,
     FlagNlRoundIcon as FlagIconNl,
-    FlagNzRoundIcon as FlagIconNz,
     FlagNoRoundIcon as FlagIconNo,
     FlagPlRoundIcon as FlagIconPl,
-    FlagPtRoundIcon as FlagIconPt,
-    FlagRoRoundIcon as FlagIconRo,
     FlagEsRoundIcon as FlagIconEs,
     FlagGbRoundIcon as FlagIconGb
 } from '@justeat/f-vue-icons';
+import { FlagSlovakiaIcon as FlagIconSk } from '@justeattakeaway/pie-icons-vue';
```

Comment:
Maybe worth importing all the flags from pie-icons-vue since we adding a new one from this package?

Reply:
hey @dandel10n, I pushed a new change. 👍